pkg/common/catalog/catalog.go

```diff
@@ -177,7 +184,13 @@ func (c *catalog) loadConfigFromHclConfig(hclPluginConfig HclPluginConfig) error
 func (c *catalog) startPlugins() error {
 	for _, p := range c.plugins {
 		if !p.Config.Enabled {
-			c.l.Debugf("%s plugin %s is disabled and will not be started", p.Config.PluginType, p.Config.PluginName)
+			c.l.Debugf("%s plugin %s is disabled", p.Config.PluginType, p.Config.PluginName)
```

Comment:
consider using k:v style logging:
c.l.Debugf("plugin_type:%s plugin plugin_name:%s is disabled", p.Config.PluginType, p.Config.PluginName)

Reply:
Since there are multiple messages "%s plugin %s is disabled" I found useful to have an indication of the context, when looking at the output log. That's why it says :  "%s plugin %s is disabled and will not be started".
E.g., we have other places where it says.: "%s plugin %s is disabled and will not be configured" and "%s plugin %s is disabled and will not be categorized".

@evan2645  Does this make sense to you?

Reply:
@amartinezfayo yes, reverted.

@walmav agreed these should be fields (logrus has first-class support for field logging). I am hesitant to change them now though since there are lots of other places in this catalog that don't use fields and the update would make these bits inconsistent. I mentioned that catalog could probably benefit from a refactor in the coming months, we should make field logging an explicit goal of that refactor, including consistent use of field names etc. I assume this is non-blocking since you approved? Will change it over if you feel strongly